schema/visibility/versioned/v0.2/reduce_open_workflow_tombstones.cql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE cadence_visibility.open_executions WITH gc_grace_seconds=60 AND compaction = {
```

Comment:
also update this file https://github.com/uber/cadence/blob/master/schema/visibility/schema.cql
It has the whole schema in one file. We have this for convenience during development

Reply:
Good catch, missed that.